src/pages/components/tabs.mdx

```diff
@@ -14,13 +14,24 @@ components:
 <Usage>
   <Use>
     <ul>
-      <li>To provide pre-filtered views of a single dataset</li>
-      <li>To organize related content and controls within a page</li>
+      <li>To filter information into easily parsable views</li>
+      <li>To organize related content and controls within a single page</li>
     </ul>
   </Use>
   <Misuse>
     <ul>
-      <li>As a secondary navigation bar that spans multiple pages</li>
```

Comment:
This doesn't seem to be encapsulated in any of the new "Not for this" items. Whether or not we're ok with it from a visual design standpoint, the `tab` component has a specific ARIA function, and can't be easily used when the different tabs take you to a different page. Why was this removed? cc @vsri1137

Reply:
Hm, I think I didn't understand this properly. That is what I hoped to capture with: 

> To help users navigate to markedly different kinds of content, use [Anchors](#) and link to a separate page.

Maybe we can replace this line with:

As a secondary navigation bar that spans multiple pages. Use Anchors instead.

What do you think?

Reply:
That's great @vsri1137 👌